pennylane/measurements/counts.py

```diff
@@ -155,7 +154,7 @@ class CountsMP(SampleMeasurement):
     Please refer to :func:`counts` for detailed documentation.
 
     Args:
-        obs (.Observable): The observable that is to be measured as part of the
+        obs (.Operator): The observable that is to be measured as part of the
```

Comment:
Why to replace the type by .Operator, we still need to pass observable? Or an composite operator that contains observables right?

Reply:
For example `Prod(PauliX(0), PauliY(1))` would be an acceptable input, but its just a plain `Operator`, not an `Observable`.